doc/cprover-manual/cbmc-unwinding.md

```diff
@@ -161,7 +161,9 @@ to the later assertion, use the option
 This option will allow paths that execute loops only partially, enabling
 a counterexample for the assertion above even for small unwinding
 bounds. The disadvantage of using this option is that the resulting path
-may be spurious, that is, it may not exist in the original program.
+may be spurious, that is, it may not exist in the original program. If
+`--unwinding-assertions` also used, however, a failing unwinding assertion
+would indicate that such spurious executions may exist.
```

Comment:
Apologies, I maybe confusing myself here... ;-)  but is the intent that if the unwinding assertion passes, then any counter examples are definitely _not_ spurious?

Reply:
Reworded!